rules/why-upgrade-to-latest-dotnet/rule.md

```diff
@@ -0,0 +1,49 @@
+---
+type: rule
+archivedreason: 
+title: Do you know why to upgrade from .NET Framework to the latest .NET?
+guid: 75e296c8-1c21-4a4e-a2fb-4df32fd6140e
+uri: why-upgrade-to-latest-dotnet
+created: 2023-10-06T23:18:38.515Z
+authors:
+  - title: Yazhi Chen
+    url: https://www.ssw.com.au/people/yazhi-chen
+
+---
+In the world of software development, staying up-to-date with technology is not just a trend; it's a necessity. As the digital landscape evolves, so do the tools and frameworks that developers rely on to build applications. One significant transition that has been taking place in recent years is the migration from the legacy .NET Framework to the latest .NET (currently .NET 7 as of November 2022 and .NET 8 will be released on November 2023) offerings. But why is this shift so crucial, and what benefits does it bring to the table?
+<!--endintro-->
+
+## The Evolution of .NET
+To understand the significance of upgrading from .NET Framework to the latest .NET, let's first take a brief look at the evolution of the .NET platform.
+
+### .NET Framework: The Legacy
+
+The .NET Framework has been the backbone of Windows application development for nearly two decades. It has provided a robust and versatile environment for building Windows desktop applications, web applications, and services. However, as technology progressed, the limitations of .NET Framework became more apparent.
+
+#### The Pain Point:
+
+* Compatibility: .NET Framework applications primarily run on Windows, limiting cross-platform deployment.
+* Performance: Older versions of .NET Framework can be resource-intensive and less efficient.
+* Security: The framework reached a point where it was no longer actively developed, leaving applications vulnerable to security risks.
+* Modern development features: .NET Framework may not support some of the modern development features and patterns that are essential for staying competitive in today's tech landscape.
+
+### Enter the Modern .NET
+Recognizing the need for change, Microsoft introduced a new direction for .NET with the release of .NET Core, which later evolved into .NET 5 and beyond. This modern .NET is designed to address the shortcomings of .NET Framework and meet the demands of contemporary software development.
+
+## Key Reasons to Upgrade
+
+* Improved Performance and Efficiency
```

Comment:
It looks like the intent here is headers, we should use h4's maybe, instead of bullets.

Also markdown like empty lines as separators so add in an extra 1 before and after the para to make sure it does the breaks as you'd expect